Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/HandConstraintPalmUp.cs

```diff
@@ -55,6 +55,14 @@ public float FlatHandThreshold
             set { flatHandThreshold = value; }
         }
 
+        [SerializeField]
+        [Tooltip("When using hybrid hand rotation, solver hand rotation until the menu is aligned with the gaze sufficiently")]
+        private bool useHybridHandRotation = false;
```

Comment:
Do you think this would make sense as an enum, in case there are other styles like this in the future (or any you can already think of)?
Or does it make more sense as something in addition to other settings?

Reply:
An enum might help, but really that would need a system review of this constraint/solver, to ensure all required patterns are handled.  It would mean the implementation would need to change to ensure it is extensible by default.

Reply:
Thanks for chiming in @SimonDarksideJ! HandConstraints in general are a bit tacked on right now, as there is no reason for them to be Hand exclusive. I think a general system review would be merited.

Beyond that, there are other conditions I can think of that would affect the rotation mode, or even the overall Solver behavior.

For example, I've written some code that involves keeping a hand menu facing the user, while it's FaceCamera vector is lined up with the GazeVector. 
In other words, even if the palm is no longer facing the camera, the menu will keep facing the camera as long as the user is gazing it. This has the benefit of keep menus stable, even when the menu is fairly large.

Implementing a feature like that would require some rethinking about how hands here.